packages/block-editor/src/components/block-list/block.js

```diff
@@ -488,6 +491,18 @@ function BlockListBlock( {
 		blockEdit = <div style={ { display: 'none' } }>{ blockEdit }</div>;
 	}
 
+	const { Fill, Slot } = createSlotFill( 'ChildToolbar' );
```

Comment:
I think we should call createSlotFill( 'ChildToolbar' ); outside of the render function, and the components ChildToolbar, ChildToolbarSlot should also be called outside too.

Reply:
I've moved outside render. Probably going to extract to a separate file.